dask/array/routines.py

```diff
@@ -170,14 +170,14 @@ def transpose(a, axes=None):
     )
 
 
-def flip(m, axis):
+def flip(m, axis=None):
     """
     Reverse element order along axis.
 
     Parameters
     ----------
```

Comment:
Perhaps also the Return block can be adjusted; since the return is not an ndarray, it should be a dask Array

```
Returns
-------
dask.array.Array
    The flipped array.
```